Cargo.lock

```diff
@@ -308,9 +315,9 @@ checksum = "6fe2267d4ed49bc07b63801559be28c718ea06c4738b7a03c94df7386d2cde46"
 
 [[package]]
 name = "hybrid-array"
-version = "0.2.0-rc.9"
+version = "0.2.0-rc.10"
```

Comment:
Might want to avoid this bump for now since it changes MSRV:

```suggestion
version = "0.2.0-rc.9"
```

Reply:
Huh, I thought `hybrid-array` was pinned by upstream.